### ISSUE TITLE
Migration from v5 to v6 with file backend

### ISSUE THREAD

**Issue by pecigonzalo** [99designs/aws-vault#663]
- [x] I am using the latest release of AWS Vault
- [x] I have provided the debug output using `aws-vault --debug` (redacted if necessary)

I just updated the CLI to `6.x` and Im getting the following error with the `file` backend.
```
aws-vault: error: exec: Failed to get credentials for wc-dev: aes.KeyUnwrap(): integrity check failed.
```

I reverted to `5.x` and it works correctly.

Debug log:
```
❯ aws-vault --debug exec dev -- aws s3 ls
2020/09/23 17:46:11 aws-vault 6.0.0
2020/09/23 17:46:11 Loading config file /home/USER/.aws/config
2020/09/23 17:46:11 Parsing config file /home/USER/.aws/config
2020/09/23 17:46:11 [keyring] Considering backends: [file]
2020/09/23 17:46:11 Profile 'personal' missing in config file
2020/09/23 17:46:11 [keyring] Expanded file dir to /home/USER/.awsvault/keys/
2020/09/23 17:46:11 [keyring] Expanded file dir to /home/USER/.awsvault/keys/
2020/09/23 17:46:11 profile personal: using stored credentials
2020/09/23 17:46:11 profile personal: skipping GetSessionToken because profile 'dev' has no MFA serial defined
2020/09/23 17:46:11 profile dev: using AssumeRole
2020/09/23 17:46:11 Looking up keyring for 'personal'
2020/09/23 17:46:11 [keyring] Expanded file dir to /home/USER/.awsvault/keys/
2020/09/23 17:46:11 [keyring] Expanded file dir to /home/USER/.awsvault/keys/
aws-vault: error: exec: Failed to get credentials for dev: aes.KeyUnwrap(): integrity check failed.
```

**Comment by mtibben** [comment]
Are you sure you're using the correct password?

**Comment by pecigonzalo** [comment]
I am, as it's set as an env var and switching the binary to version 5 makes
it work again without any other changes.
I also tried removing the env var and passing it manually when the CLI
prompts and no dice.

**Comment by mtibben** [comment]
Fixed in f25873e4d5d5c8ed30694e8434d183825922c934

**Comment by phyber** [comment]
This fix appears to have broken things for me when going from 6.1.0 to 6.2.0.
Is this because `file` backends created with 6.x were broken before 6.2.0?

Sorry for replying to a closed issue, I wasn't sure if this should be reported in a new issue or not.
